apply2.html

```diff
@@ -0,0 +1,418 @@
+---
+layout: default
+permalink: "/apply/2"
+redirect_from:
+- "/application"
+head_title: Start Your Application | Apply Now at {{ school.name }}
+description: Apply now to learn to code with {{ school.name }} | Fill out
+  your short application to get on the path to a career in web development.
+
+form_fields:
+- name: email
+  label: ''
+  description: ''
+  type: hidden
+  value: ''
+  placeholder: ''
+  required: ''
+  small_width: ''
+  medium_width: ''
+  options: []
+- name: application_submission_url
+  label: ''
+  description: ''
+  type: hidden
+  value: "{{ school.url }}{{ site.baseurl }}{{ page.permalink }}"
+  placeholder: ''
+  required: ''
+  small_width: ''
+  medium_width: ''
+  options: []
+- name: lifecyclestage
+  label: ''
+  description: ''
+  type: hidden
+  value: opportunity
+  placeholder: ''
+  required: ''
+  small_width: ''
+  medium_width: ''
+  options: []
+- name: filled_out_application
+  label: ''
+  description: ''
+  type: hidden
+  value: true
+  placeholder: ''
+  required: ''
+  small_width: ''
+  medium_width: ''
+  options: []
+- name: city
+  label:  Current City
+  description: ''
+  type: text
+  value: ''
+  placeholder: City
+  required: true
+  small_width: "12"
+  medium_width: "6"
+  options: []
+- name: state
+  label: Current State
+  description: ''
+  type: select
+  value: ''
+  placeholder: State
+  required: true
+  small_width: "12"
+  medium_width: "6"
+  options:
```

Comment:
We can probably just let them type in the state abbreviate here so we don't have to maintain this list of states

Reply:
@SeanPhilippi